src/main/java/org/folio/service/CustomFieldsServiceImpl.java

```diff
@@ -1,34 +1,48 @@
 package org.folio.service;
 
+import static io.vertx.core.Future.failedFuture;
+import static io.vertx.core.Future.succeededFuture;
+
 import java.text.Normalizer;
 
 import io.vertx.core.Future;
-
-import org.folio.rest.jaxrs.model.CustomFieldCollection;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
+import org.folio.common.OkapiParams;
 import org.folio.repository.CustomFieldsRepository;
 import org.folio.rest.jaxrs.model.CustomField;
+import org.folio.rest.jaxrs.model.CustomFieldCollection;
 import org.folio.service.exc.ServiceExceptions;
 
 @Component
 public class CustomFieldsServiceImpl implements CustomFieldsService {
 
   @Autowired
   private CustomFieldsRepository repository;
+  @Autowired
+  private UserService userService;
 
   @Override
-  public Future<CustomField> save(CustomField customField, String tenantId) {
-    final String customFieldId = unAccentName(customField.getName());
+  public Future<CustomField> save(CustomField customField, OkapiParams params) {
+    final String unAccentName = unAccentName(customField.getName());
 
```

Comment:
@carolegodfrey we use the name of the field to create a reference id, which in our case is actually id field to give the user a way to operate with a meaningful name instead of id. Also, this refId field is planned to be included in the export/import feature.